web/hooks/useRandomShape.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react';
+import type { Shape } from '../types/Shape';
+import styles from './shapes.module.css';
+
+const shapes: Shape[] = ['Plus', 'C', 'Ovals', 'O', 'HalfOval'];
+
+const getRandomShape = () => shapes[Math.floor(Math.random() * shapes.length)];
+
+export const useRandomShape = (randomShapeChance: number = 1) => {
+  const [shapeClass, setShapeClass] = useState<string>();
+
+  useEffect(() => {
+    if (!shapeClass && Math.random() <= randomShapeChance) {
+      setShapeClass(styles[`shape${getRandomShape()}`]);
+    }
+  }, [shapeClass, randomShapeChance]);
+
+  return shapeClass;
```

Comment:
Nit: Is is possible to make this return a string (or null) instead in such a way that all the `.shape` styles are collected in one place instead of being split across two separate files (one in the hooks directory and one in the usual/expected location in the components directory)?

Reply:
I don't know why I struggled with de-duplicating the shapes' classes before, but that's why they were left like that. A bit confused as to why, because converting _FakeItem.tsx_ to use the new hook was hassle-free: a4e9a63